test/integration/model/find.test.js

```diff
@@ -1024,18 +1024,33 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         username: Sequelize.STRING
       }, { paranoid: true });
 
-      return User.sync({ force: true }).then(() => {
-        return User.bulkCreate([
-          {username: 'Bob'},
-          {username: 'Tobi'}
-        ]);
-      }).then(() => {
-        return User.destroy({ where: {username: 'Tobi'} });
-      }).delay(5000).then(() => {
-        return User.findAll({ paranoid: true });
-      }).then(users => {
-        expect(users.length).to.be.eql(1);
-      });
+
+      if (dialect === 'sqlite') 
```

Comment:
@felixfbecker  Done (https://github.com/sequelize/sequelize/pull/7787/commits/81590619c65b3cbf89ea2abe089e70a1a72859ea) and passed on my test (https://travis-ci.org/getglad/sequelize/builds/246334145).